src/main/java/org/matsim/prepare/CreateSupply.java

```diff
@@ -170,12 +173,22 @@ private void run(Path rootDirectory) {
 		new NetworkWriter(network3).write(rootDirectory.resolve(outputDir.resolve("metropole-ruhr-v1.0.network-onlyBikeNetwork3.xml.gz")).toString());
 		new BikeNetworkMerger(network).mergeBikeHighways(network3);
 
+		//-------------------------- add height information to network -------------------------------------------------
+
+		//TODO get correct transformation
```

Comment:
I need to know the crs of the geo-tif, so that we can insert the correct transformation here. Should be geo-tif-crs -> UTM-32